Bootstrap/Update.php

```diff
@@ -223,4 +224,18 @@ private function clearConfigCache()
             $cacheManager->clearConfigCache();
         }
     }
+
+    private function updateConnectAttribute(){
+        if (version_compare($this->version, '1.0.6', '<=')) {
```

Comment:
We have to discuss about versions and different branches. It's possible to have another branch with update.